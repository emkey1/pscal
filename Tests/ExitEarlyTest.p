program ExitEarlyTest;

function Foo(): integer;
begin
  writeln('Inside Foo');
  exit;
  Foo := 1; { This line should not execute }
end;

begin
  writeln('Start');
  Foo;
  writeln('After Foo');
end.
