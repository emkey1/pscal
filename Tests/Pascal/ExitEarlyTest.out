Start
Inside Foo
After Foo
